File: Lab1/Lab1/Lab1/db/SysteUnitRepository.cs
using Lab1.dto;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Lab1.dto;

namespace Lab1.db
{
    public class SysteUnitRepository
    {
        public static List<SystemUnitDto> findAll()
        {
            List<SystemUnitDto> systemUnits = new List<SystemUnitDto>();//спросить про ИДЕЕ!!!
            systemUnits.Add(new SystemUnitDto(MotherboardTypes.BIG,1000,null,null,"BIG"));
            systemUnits.Add(new SystemUnitDto(MotherboardTypes.BIG, 100, null, null, "BIG"));
            systemUnits.Add(new SystemUnitDto(MotherboardTypes.BIG, 100, null, null, "BIG"));
            return systemUnits;
        }
    }
}

File: Lab1/Lab1/lab1j/src/db/SysteUnitRepository.java
package db;

import dto.MotherboardTypes;
import dto.SystemUnitDto;

import java.util.LinkedList;
import java.util.List;

public class SysteUnitRepository
    {
        public static List<SystemUnitDto> findAll()
        {
            List<SystemUnitDto> systemUnits = new LinkedList<>();//спросить про ИДЕЕ!!!
            systemUnits.add(new SystemUnitDto(MotherboardTypes.BIG,1000,null,null,"BIG"));
            systemUnits.add(new SystemUnitDto(MotherboardTypes.BIG, 100, null, null, "BIG"));
            systemUnits.add(new SystemUnitDto(MotherboardTypes.BIG, 100, null, null, "BIG"));
            return systemUnits;
        }
    }

